constructs/Compute/virtualMachinesMultiple/main.bicep

```diff
@@ -242,13 +242,8 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 @description('Optional. Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.')
 param diagnosticEventHubName string = ''
 
-@allowed([
-  ''
-  'CanNotDelete'
-  'ReadOnly'
-])
-@description('Optional. Specify the type of lock.')
-param lock string = ''
+@description('Optional. The lock settings of the service.')
+param lock lockType
```

Comment:
Does this work without importing the type? as the type is not defined in this file?

Reply:
It does not. Good catch. Led me revoke all constructs. Must have been a search and replace issue

Reply:
done